Example/PaymentSheet Example/PaymentSheet Example/PlaygroundController.swift

```diff
@@ -1009,6 +1018,57 @@ extension PlaygroundController {
             }
         }
     }
+
+    func createIntent() async throws -> String {
+        return try await withCheckedThrowingContinuation { continuation in
+            var body = [
+                "currency": settings.currency.rawValue,
+                "amount": settings.amount.rawValue,
+                "merchant_country_code": settings.merchantCountryCode.rawValue,
+                "mode": settings.mode.rawValue,
+                "automatic_payment_methods": settings.apmsEnabled == .on,
+                "use_link": settings.linkPassthroughMode == .pm,
+                "link_mode": settings.linkEnabledMode.rawValue,
+                "use_manual_confirmation": settings.integrationType == .deferred_mc,
+                "require_cvc_recollection": settings.requireCVCRecollection == .on,
+                "is_confirmation_token": settings.confirmationMode == .confirmationToken && !settings.integrationType.isIntentFirst,
+                "payment_method_options_setup_future_usage": settings.paymentMethodOptionsSetupFutureUsage.toDictionary(),
```

Comment:
What's the reasoning for omitting the customer session in this payload compared to the loadBackend?